core/src/main/java/io/ddf/content/Schema.java

```diff
@@ -398,24 +398,32 @@ public void setData(Object[] data) {
 
 
   /**
-   * The R concept of a column "type", such as STRING, INT, LONG, etc.
+   * The R concept of a column "type", such as STRING, INT, BIGINT, etc.
    */
   public enum ColumnType {
```

Comment:
I think that would be better approach since different engines support different types. It will keep the DDF API cleaner and easier for supporting other engines.